pkg/plugins/golang/v4/scaffolds/internal/templates/makefile.go

```diff
@@ -100,23 +100,23 @@ help: ## Display this help.
 
```

Comment:
If you change the bollerplate than you need to run make generate to ensure that you will update all samples with under the testdata.

Also, is the problem that you are trying to solve here faced with the default scaffold or only if you do some specific manually change on it?